src/core/tUnification.ml

```diff
@@ -432,7 +432,7 @@ let direct_access = function
 	| AccNo | AccNever | AccNormal | AccInline | AccRequire _ | AccCtor -> true
 	| AccCall -> false
 
-let unify_kind ?(strict:bool = false) k1 k2 =
+let unify_kind ~(strict:bool) k1 k2 =
```

Comment:
What I meant was simply making it a required argument, but it's fine, maybe I'll randomly change that later!

Reply:
Yeah but random booleans are a bit weird :/